data/scenarios/scenarios.json

```diff
@@ -0,0 +1,10 @@
+"scenarios": [
+    {
```

Comment:
I gave two AMI IDs in slack.  One CentOS and the other Ubuntu.  Figured people would choose whatever they were most comfortable with.  I didn't think about providing other broken images.